pkg/services/live/managedstream/runner_test.go

```diff
@@ -55,7 +55,7 @@ func TestGetManagedStreams(t *testing.T) {
 
 	managedChannels, err := runner.GetManagedChannels(1)
 	require.NoError(t, err)
-	require.Len(t, managedChannels, 4) // 4 hardcoded testdata streams.
+	require.Len(t, managedChannels, 9) // 9 hardcoded testdata streams.
```

Comment:
Are these changes in tests necessary here? I think in normal test run we won't have those additional channels?

Reply:
no longer there in https://github.com/grafana/grafana/pull/43295